packages/contracts/src/library/Utils.sol

```diff
@@ -203,4 +203,12 @@ library Utils {
         }
         Game.popPlayers(_gameId);
     }
+
+    function generateRandomAddress(address _player) internal returns (address randomAddr) {
```

Comment:
It's not random, it's deterministic. At least add `blochhash(block.number-1)` or `gasleft()`

Reply:
Get the player's fixed enemy address

Reply:
Use another accurate name instead